test/network/NetworkSettings.ts

```diff
@@ -211,11 +221,13 @@ describe('NetworkSettings', () => {
 
             it('should remove a token', async () => {
                 expect(await networkSettings.isTokenWhitelisted(reserveToken.address)).to.be.true;
+                expect(await networkSettings.protectedTokenWhitelist()).to.include(reserveToken.address);
 
                 const res = await networkSettings.removeTokenFromWhitelist(reserveToken.address);
                 await expect(res).to.emit(networkSettings, 'TokenRemovedFromWhitelist').withArgs(reserveToken.address);
 
                 expect(await networkSettings.isTokenWhitelisted(reserveToken.address)).to.be.false;
+                expect(await networkSettings.protectedTokenWhitelist()).not.to.include(reserveToken.address);
```

Comment:
Semantic: I'd use `to.not.include` (3 places in this file, one of which is not part of this PR).

Reply:
Sure

Reply:
Actually, it works either way, so let's not waste time on this